lib/fastifySession.js

```diff
@@ -18,8 +18,8 @@ function session (fastify, options, next) {
   fastify.decorate('decryptSession', (sessionId, request, callback) => {
     decryptSession(sessionId, options, request, callback)
   })
-  fastify.decorateRequest('sessionStore', options.store)
-  fastify.decorateRequest('session', {})
+  fastify.decorateRequest('sessionStore', { getter: () => options.store })
```

Comment:
I am not sure about it. The `store` can be changed before but it cannot change now.

If it is `getter` only, the type should be changed to `readonly`.

Reply:
Hey @zekth, any comment on this ?

Reply:
Indeed this is a readonly value, please adjust the typings

Reply:
Hey, @zekth, I am not sure about what you are talking about?
Will you suggest to me the changes that need to do?

Reply:
@vikasg603
Change it to readonly. https://github.com/fastify/session/blob/master/types/types.d.ts#L10
Add test-case inside https://github.com/fastify/session/blob/master/types/test/using-store.ts

It should be something like this, we expect it to be throwing error
```ts
// @ts-expect-error
request.sessionStore = null
```